localstack/services/awslambda/lambda_executors.py

```diff
@@ -305,13 +308,9 @@ def prepare_execution(self, func_arn, env_vars, runtime, command, handler, lambd
         # determine files to be copied into the container
         copy_command = ''
         docker_cmd = self._docker_cmd()
-        event_file = os.path.join(lambda_cwd, LAMBDA_EVENT_FILE)
         if not has_been_invoked_before and config.LAMBDA_REMOTE_DOCKER:
             # if this is the first invocation: copy the entire folder into the container
             copy_command = '%s cp "%s/." "%s:/var/task";' % (docker_cmd, lambda_cwd, container_info.name)
-        elif os.path.exists(event_file):
-            # otherwise, copy only the event file if it exists
-            copy_command = '%s cp "%s" "%s:/var/task";' % (docker_cmd, event_file, container_info.name)
```

Comment:
Not sure what was the reason to remove this, but this broke the docker-reuse mode. It works fine on the first invocation but when a container is reused on subsequent invocations, the event_file is not being copied to the lambda container causing the invocation to fail:

```
Exception in thread "main" java.nio.file.NoSuchFileException: /var/task/_lambda.events.5f35aa33.json
	at java.base/sun.nio.fs.UnixException.translateToIOException(Unknown Source)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(Unknown Source)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(Unknown Source)
	at java.base/sun.nio.fs.UnixFileSystemProvider.newFileChannel(Unknown Source)
	at java.base/java.nio.channels.FileChannel.open(Unknown Source)
	at java.base/java.nio.channels.FileChannel.open(Unknown Source)
	at java.base/java.nio.file.Files.lines(Unknown Source)
	at cloud.localstack.LambdaExecutor.readFile(LambdaExecutor.java:171)
	at cloud.localstack.LambdaExecutor.main(LambdaExecutor.java:55) Traceback (most recent call last):
```

On docker mode, it works fine. I'd raise a PR but don't quite understand the new code around this and how the event is being copied now.